release-notes/docs/3.5.0.md

```diff
@@ -0,0 +1,179 @@
+# 3.5.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Performance](#performance)
+- [Sort muted contacts](#sort-muted-contacts)
+- [SIH integration](#sih-integration)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+There are no breaking changes when upgrading from `3.4.x`.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## Performance
+
+### Optimizing the History tab
+
+We found that loading the History tab was taking a long time in projects with many places. This was due to loading the Places filter on the tab. By delaying the initialisation of the filter until needed we found performance improvements in three key areas. [medic#4423](https://github.com/medic/medic/issues/4423).
+
+| Metric | 3.4.0 | 3.5.0 | Improvement |
+|---|---|---|---|
+| Rendering the History tab | 7247ms | 1843ms | 75% |
+| Starting the application | 1020ms | 875ms | 14% |
+| Idle memory usage | 453MB | 202MB | 55% |
+
+A further optimization was made to the History detail pane by improving how we link to Patients, which reduced rendering time from 1002ms to 732ms an improvement of 27%. [medic#5455](https://github.com/medic/medic/issues/5455)
+
+### Sentinel processing
+
+By reducing the the throttling of change processing and reducing the number of database writes we have greatly improved the rate at which Sentinel processes changes. Clearing a backlog of 200 documents used to take 67 seconds, but with 3.5.0 this has been reduced to 33 seconds, a 50% reduction. [medic#4128](https://github.com/medic/medic/issues/4128).
+
+To make Sentinel even more responsive to changes, transitions are now executed synchronously when messages are received. This means that replies are generated immediately and the user is be notified of the success or error of their message as soon as possible. [medic#4857](https://github.com/medic/medic/issues/4857).
```

Comment:
typo `the user is be notified`
instead of `immediately` maybe `faster`

Reply:
I think it's correct to say "immediately". It's synchronous so immediately after the message is received we generate the response and save the doc. The bit that's merely faster is the notification being sent to the user...

Reply:
Typo fixed though! Thanks.